lib/marklogic.js

```diff
@@ -776,9 +778,8 @@ function initClient(client, inputParams) {
     client.request = https.request;
     if (noAgent) {
       mlutil.copyProperties(inputParams, agentOptions, [
-        'ca', 'cert', 'ciphers', 'keepAliveMsecs', 'key', 'maxSockets',
-        'maxFreeSockets', 'passphrase', 'pfx', 'rejectUnauthorized',
-        'secureProtocol', 'timeout'
+        'keepAliveMsecs', 'maxSockets', 'maxTotalSockets',
+        'maxFreeSockets', 'timeout', 'scheduling'
```

Comment:
As stated in the issue, I believe that the agent options for SSL HTTPS should be:

keepAliveMsecs, maxCachedSessions, maxFreeSockets, maxSockets, maxTotalSockets, scheduling, timeout